docs/cookbook/cloudways.rst

```diff
@@ -0,0 +1,93 @@
+How to deploy Sylius to Cloudways PHP Hosting?
+==============================================
+
+Cloudways is a managed hosting platform for custom PHP apps and PHP frameworks such as Symfony, Laravel, Codeigniter, Yii
+CakePHP etc. You can launch the servers on any of the five providers including DigitalOcean, Vultr, AWS, GCE and KYUP containers.
+
+The deployment process of Sylius on Cloudways is pretty much straight forward and easy.
+
+Now to install Sylius you need to go through series of few steps:
+
+1. Launch Server with Custom PHP App
+------------------------------------
+
+You should `signup at Cloudways <https://platform.cloudways.com/signup>`_ to buy the PHP servers from the above mentioned providers. Simply go to the pricing page and choose your required plan.
+You then need to go through the verification process. Once it done login to platform and launch your first Custom PHP application. You can follow the Gif too.
+
+.. image:: ../_images/cloudways-php-server.gif
+    :align: center
+
+Now let’s start the process of installing Sylius on Cloudways.
+
+2. Install Sylius v1.0.0 via SSH
```

Comment:
We can skip the version as you'd like to install the latest one, not excatly `1.0.0`